native/src/components/FeedbackContainer.tsx

```diff
@@ -99,7 +111,7 @@ const FeedbackContainer = (props: FeedbackContainerProps): ReactElement => {
       city: cityCode,
       language,
       comment: commentWithMail,
-      query,
+      query: searchTerm?.trim().length === 0 ? query : searchTerm,
```

Comment:
Actually I think it might make sense to just send both (similar to what we do with the contact address:
```suggestion
      query: query === searchTerm ? query : `${searchTerm} (actual query: ${query})`,
```

That would give us/the cities even more information to work with and probably also an insight if the current UI is misleading/confusing.

Reply:
That's a great idea!